lib/tasks/test_suite.rake

```diff
@@ -0,0 +1,14 @@
+desc 'Run all tests in application-defined order'
+task :test_suite => :environment do
+  environment_protected
+  Rake::Task['rubocop'].invoke
+  Rake::Task['jasmine:ci'].invoke
+  Rake::Task['spec'].invoke
+  system('printf "\e[33mInfo: Sleeping for five seconds to give CPU time to cool down and perhaps not fail on the cuke tasks because drop down lists aren''t populated fast enough\e[0m"')
```

Comment:
Does this line need a newline too? You've added `\n`  to all the other `printf`s...